src/Tickets/Commerce/Gateways/PayPal/REST/Order_Endpoint.php

```diff
@@ -177,7 +177,9 @@ public function handle_update_order( WP_REST_Request $request ) {
 			return new WP_Error( 'tec-tc-gateway-paypal-nonexistent-order-id', null, $order );
 		}
 
-		$paypal_capture_response = tribe( Client::class )->capture_order( $paypal_order_id );
+		$order_data = (array) json_decode( $request->get_body() );
```

Comment:
We should use `get_param()` from the request and pick each item individually. To avoid the camel casing.